andela/ah-backend-zeus
# #162163166 User account verification via email

## What does this PR do?
- The PR adds a feature that enables successfully registered users to verify their accounts via email.
### Description of task to be completed?
- A user must click a link sent to their email after registration to verify their account.
### How should this be manually tested?
- Clone this repository and checkout the branch: 'ft-email-verification-link-162163166'.
- Set up the project.
- Register a new user with a valid email address with Postman.
- Check the email and click the verification link.
### What are the relevant pivotal tracker stories?
[#162163166](https://www.pivotaltracker.com/story/show/162163166)
